fixtures/manufacturers.json

```
@@ -180,6 +180,11 @@
   "evolight": {
     "name": "Evolight"
   },
+  "exalux": {
+    "name": "EXALUX",
+    "website": "https://www.exalux.eu/",
+    "rdmId": 84
```

```suggestion
    "rdmId": 132
```

According to [this list](https://tsp.esta.org/tsp/working_groups/CP/mfctrIDs.php), "LEDIXIS (Exalux brand)" is 0084h, which is 132 in decimal.